projects/plugins/jetpack/tests/e2e/specs/pre-connection.test.js

```diff
@@ -22,7 +22,7 @@ describe( 'Jetpack pre-connection', () => {
 
 	afterAll( async () => {
 		await execWpCommand(
-			`wp option update jetpack_private_options --format=json < ${ path.resolve(
+			`option update jetpack_private_options --format=json < ${ path.resolve(
```

Comment:
This breaks any following test that requires a connection. It adds the blog token back making the tests think there's an active connection but there isn't one. It doesn't seem to work in combination with `withCleanEnv()` in `beforeAll`. I think it should be removed entirely.

Reply:
Good catch! But I guess this is not related to this PR. Removed in 9e05ad5